configs/_base_/datasets/pipelines/auto_aug.py

```diff
@@ -0,0 +1,104 @@
+sub_policy_0 = [
+    dict(type='Posterize', bits=8, prob=0.4),
+    dict(type='Rotate', angle=9., prob=0.6)
+]
+sub_policy_1 = [
+    dict(type='Solarize', thr=5, prob=0.6),
+    dict(type='AutoContrast', prob=0.6)
+]
+sub_policy_2 = [
+    dict(type='Equalize', prob=0.8),
+    dict(type='Equalize', prob=0.6)
+]
+sub_policy_3 = [
+    dict(type='Posterize', bits=7, prob=0.6),
+    dict(type='Posterize', bits=6, prob=0.6)
+]
+sub_policy_4 = [
+    dict(type='Equalize', prob=0.4),
+    dict(type='Solarize', thr=4, prob=0.2)
+]
+sub_policy_5 = [
+    dict(type='Equalize', prob=0.4),
+    dict(type='Rotate', angle=8., prob=0.8)
+]
+sub_policy_6 = [
+    dict(type='Solarize', thr=3, prob=0.6),
+    dict(type='Equalize', prob=0.6)
+]
+sub_policy_7 = [
+    dict(type='Posterize', bits=5, prob=0.8),
+    dict(type='Equalize', prob=1.0)
+]
+sub_policy_8 = [
+    dict(type='Rotate', angle=3., prob=0.2),
+    dict(type='Solarize', thr=8, prob=0.6)
+]
+sub_policy_9 = [
+    dict(type='Equalize', prob=0.6),
+    dict(type='Posterize', bits=6, prob=0.4)
+]
+sub_policy_10 = [
+    dict(type='Rotate', angle=8., prob=0.8),
+    dict(type='ColorTransform', magnitude=0, prob=0.4)
+]
+sub_policy_11 = [
+    dict(type='Rotate', angle=9., prob=0.4),
+    dict(type='Equalize', prob=0.6)
+]
+sub_policy_12 = [
+    dict(type='Equalize', prob=0.0),
+    dict(type='Equalize', prob=0.8)
+]
+sub_policy_13 = [
+    dict(type='Invert', prob=0.6),
+    dict(type='Equalize', prob=1.0)
+]
+sub_policy_14 = [
+    dict(type='ColorTransform', magnitude=4, prob=0.6),
+    dict(type='Contrast', magnitude=8, prob=1.0)
+]
+sub_policy_15 = [
+    dict(type='Rotate', angle=8., prob=0.8),
+    dict(type='ColorTransform', magnitude=2, prob=1.0)
+]
+sub_policy_16 = [
+    dict(type='ColorTransform', magnitude=8, prob=0.8),
+    dict(type='Solarize', thr=7, prob=0.8)
+]
+sub_policy_17 = [
+    dict(type='Sharpness', magnitude=7, prob=0.4),
+    dict(type='Invert', prob=0.6)
+]
+sub_policy_18 = [
+    dict(type='Shear', magnitude=5, prob=0.6),
+    dict(type='Equalize', prob=1.0)
+]
+sub_policy_19 = [
+    dict(type='ColorTransform', magnitude=0, prob=0.4),
+    dict(type='Equalize', prob=0.6)
+]
+sub_policy_20 = [
+    dict(type='Equalize', prob=0.4),
+    dict(type='Solarize', thr=4, prob=0.2)
+]
+sub_policy_21 = [
+    dict(type='Solarize', thr=5, prob=0.6),
+    dict(type='AutoContrast', prob=0.6)
+]
+sub_policy_22 = [
+    dict(type='Invert', prob=0.6),
+    dict(type='Equalize', prob=1.0)
+]
+sub_policy_23 = [
+    dict(type='ColorTransform', magnitude=4, prob=0.6),
+    dict(type='Contrast', magnitude=8, prob=1.0)
+]
+
+auto_increasing_policies = [
```

Comment:
Maybe reuse autoaug policies style in [#395](https://github.com/open-mmlab/mmclassification/blob/ac0bba5ec164633c80264e589a7208ebe47a2473/configs/_base_/datasets/pipelines/auto_aug.py)?
Will these sub-policies be reused in other policies? If not, it’s not necessary to split them to multiple variables.

Reply:
OK.

Reply:
since there is no official auto_aug in repvgg repo and training accuracy is not aligned, so use auto_aug in [#395](https://github.com/open-mmlab/mmclassification/blob/ac0bba5ec164633c80264e589a7208ebe47a2473/configs/_base_/datasets/pipelines/auto_aug.py) temporarily.